src/themes/material/modules/dropdown.overrides

```diff
@@ -1,4 +1,4 @@
-@import url(https://fonts.googleapis.com/css?family=Roboto:400,700);
+@import (css) url('https://fonts.googleapis.com/css2?family=Roboto');
```

Comment:
Forgot [weight range](https://developers.google.com/fonts/docs/css2#axis_ranges) ? 
```suggestion
@import (css) url('https://fonts.googleapis.com/css2?family=Roboto:wght@400..700');
```

Reply:
Only weight 400 is available in the v2 api, that's why i removed the weight completely.
Your suggestions results in an unresolvable link:
https://fonts.googleapis.com/css2?family=Roboto:wght@400..700
![image](https://user-images.githubusercontent.com/18379884/146695424-52e7b554-40a5-4b1c-b230-6877d65e8fee.png)